core/src/main/java/org/apache/accumulo/core/security/ColumnVisibility.java

```diff
@@ -598,6 +600,7 @@ public Node getParseTree() {
    * @param term term to quote
    * @return quoted term (unquoted if unnecessary)
    */
+  // TODO deprecate?
```

Comment:
maybe add why to explain later when someone comes across this?

`// TODO deprecate in favor of X when X is implemented`  or ` // TODO deprecate when unused because replaced with ...`

Reply:
I am trying to decide if that method should be deprecated.  I made a comment about this on #4206

Reply:
I think if we were going to make AccessExpression the main way to create a ColumnVisibility in the future, such that the user had to create one and pass it to the ColumnVisibility, then I would say yes, deprecate this.

Reply:
deprecated in c88457a